pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -182,7 +182,7 @@ func nodeToStartRevisionWith(getStaticPodState func(nodeName string) (state stat
 
 // manageInstallationPods takes care of creating content for the static pods to install.
 // returns whether or not requeue and if an error happened when updating status.  Normally it updates status itself.
-func (c *InstallerController) manageInstallationPods(operatorSpec *operatorv1.OperatorSpec, originalOperatorStatus *operatorv1.StaticPodOperatorStatus, resourceVersion string) (bool, error) {
+func (c *InstallerController) manageInstallationPods(operatorSpec *operatorv1.StaticPodOperatorSpec, originalOperatorStatus *operatorv1.StaticPodOperatorStatus, resourceVersion string) (bool, error) {
```

Comment:
doesn't look like this needed to change

Reply:
we call this function [here](https://github.com/openshift/library-go/blob/master/pkg/operator/staticpod/controller/installer/installer_controller.go#L519) which needs the `StaticPodOperatorStatus` to be able to loop through the `NodeStatuses`. Since `GetStaticPodOperatorState` now returns the `StaticPodOperatorSpec` along with the status it looks like we need to carry that through